.github/workflows/self-push.yml

```diff
@@ -78,6 +78,60 @@ jobs:
           name: run_all_tests_torch_gpu_test_reports
           path: reports
 
+  run_tests_flax_gpu:
+    runs-on: [self-hosted, docker-gpu, single-gpu]
+    container:
+      image: tensorflow/tensorflow:2.4.1-gpu
```

Comment:
(Nitpick, if you're good with this image then let's got with this image)

What does the TensorFlow image provide that is helpful for the JAX setup? I'm wondering if a more lightweight image cannot be taken - for example if the only package of interest is `cuda`, maybe leveraging an official image from NVIDIA with only `cuda` installed would be sufficient?

An image such as this one: `docker pull nvcr.io/nvidia/cuda:11.4.1-devel-ubi8` from [here](https://ngc.nvidia.com/catalog/containers/nvidia:cuda).

Reply:
Will try this one out

Reply:
Not really sure why TensorFlow works here out of the box...The image `docker pull nvcr.io/nvidia/cuda:11.4.1-devel-ubi8` doesn't work though :-/ Is it problematic to use the Tensorflow image here? Think I'd opt for this at the moment until an official JAX image is out